apps/discord-bot/utils/scheduledEventFunctions.ts

```diff
@@ -12,7 +12,7 @@ export const createLiveStreamEventFromMessage = (
   message: Message,
   params: ICreateLiveStreamEventParams,
 ) => {
-  if (!message.guild) return;
+  if (!message.guild || !message.author.bot) return;
```

Comment:
Can you explain what this condition checks exactly? I understand the `!message.author.bot` part, but not sure about the `!message.guild` part. Shouldn't we check that value against a guild id or something?

Reply:
this line is caused by the fact that one type of the guild object coming from the message object in the discord.js library is also null